sbt-metals/src/main/scala/MetalsPlugin.scala

```diff
@@ -1,91 +1,91 @@
+package scala.meta.sbt
+
 import sbt._
 import sbt.Keys._
 import java.io._
-import sbt.Def
 
 object MetalsPlugin extends AutoPlugin {
   override def trigger = allRequirements
   override def requires = sbt.plugins.JvmPlugin
   object autoImport {
 
     val metalsCompilerConfig =
-      taskKey[String](
+      taskKey[Seq[(String, String)]](
         "String containing build metadata in properties file format."
```

Comment:
> List of key/value pairs for build information such as classpath/sourceDirectories.

Reply:
Again, this refers to an outdated diff (sorry for the mess)